Branch-TestBed/ANSCompatibility.h

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
I think we need them but I should move them to the Fabric subdirectory under Branch-SDK